packages/pwa-kit-extension-sdk/src/configs/babel/plugin-application-extensions.ts

```diff
@@ -13,7 +13,7 @@ const babel = require('@babel/core')
 
 // Local
 import {renderTemplate} from '../utils'
-import {buildAliases} from '../../shared/utils'
+import {buildAliases, validateExtensionDependencies} from '../../shared/utils'
```

Comment:
I like that you create this utility in a way that it's reusable. Which might make this suggestion something easier to do.

When I ran your scenario's I found that it was really hard to, at a glance, see why the build or server start failed. Because it got pretty much buried in  a bunch of other mess with a stack trace and what not. 

Can you thought about moving this validation into the pwa-kit-dev? So before we start the dev server or build process we can validate the configuration and simply not kick off the server if something is wrong. You might end up with a simply output like below if we do something like that:

```
> npm run start
> --> Starting...
> --> Validating configuration...
> --> Checking Extension Dependencies...
> [\]
> Error: Extension(s) missing or disabled: @salesforce/extension-sample, as required by @salesforce/extension-sample-2
> Please check your configuration for errors ⚠️
```

I think doing something like that will make it easier for the end developer to understand whats wrong at a glance. And we can also use this as a starting point for doing other things. Like if we have configuration validation for extensions, or the base app itself.